api/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java

```diff
@@ -294,9 +294,136 @@ public final class NessieConfigConstants {
    */
   @Deprecated
```

Comment:
BTW/FYI: There's a javadoc tag (or annotation?) to hide this, if you want to.

Reply:
Or do we want to rewrite it automatically to the new name?

Reply:
To hide what?

About automatic remapping: I thought about it but I prefer not to. The settings have similar names but completely different behaviors.

Reply:
The setting in the generated docs.